tequila_fab/__init__.py

```diff
@@ -37,7 +42,7 @@ def bootstrap():
     install_roles()
     execute(check_role_versions)
     deploy("bootstrap_python")
-    deploy("site", extra_vars='{"unmanaged_users": [ubuntu]}')
```

Comment:
Haven't done a full review yet, but noting this for @JeremyGibson 

This was used to not lockout the user deploying during an initial provision, which can happen if a playbook fails mid-run the first time. If we remove this, it might be nice to allow passing it into ``bootstrap`` somehow. Alternatively, I suppose projects that do want this could just use its own ``bootstrap`` copy and not call this task.

Reply:
@copelco @JeremyGibson 
I removed `extra_vars='{"unmanaged_users": [ubuntu]}'` because I thought Tequila should be agnostic about whether we're deploying to an AWS ubuntu system which starts with an ubuntu user, a Vagrant system which starts with a vagrant user, a DigitalOcean system which might start with the dev's account already ssh accessible, etc.

Alternatively we could try to cover as many options as possible here, I suppose.

Reply:
I'm disturbed this change was made in an apparently unrelated ticket. Did something about role-checking require this change be made? If not I'd prefer things be put back the way they were and a new ticket opened to discuss the merits of being inclusive or agnostic or whatever here...

Reply:
New release undoes this change for now. https://github.com/caktus/tequila-fab/releases/tag/0.0.4